### DISCUSSION TITLE
Specify backup folder by code

### DISCUSSION THREAD

**Discussion by sksar** [pocketbase/pocketbase#6174] [Q&A] [answered]
I am extending PocketBase using the Go programming language framework.
I am seeking help to configure the backup folder location.

Currently, the backup folder is located within the pb_data folder. 
However, I would like to configure it to its own folder, such as pb_backup.

Could someone please provide guidance on how to accomplish this?

@ganigeorgiev

**Comment by ganigeorgiev** [comment, marked as answer]
_Please don't ping me for no reason. I cannot provide dedicated support (see [FAQ](https://pocketbase.io/faq/)). Additionally it also discourage other users to join the discussion and eventually assist you._

In any case, similar to the storage directory, the default backup directory at the moment is not configurable. We can add an option for that but personally I don't see the reason for it (its behavior is also unclear as it could cause "cross-device link errors" in case the backup is stored on a different disk). Just because someone wants to be able to do it is not enough to convince me to dedicate time on this.

At least on linux, as a workaround you should be able to mount any directory, including network mounted one, in `pb_data/backups` manually (_be aware of https://github.com/pocketbase/pocketbase/issues/4647#issuecomment-2543003758_).

If it represents a blocker and a problem for your use case, feel free to elaborate but for now this remains out of the scope.
